controls/radrichtextbox/features/merge-documents/merge.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Insert Document at Caret Position
+page_title: Insert Document at Caret Position
+description: Insert Document at Caret Position
+slug: radrichtextbox-features-merge-insert-document
+tags: raddocumentmerger, merge, insert, documents
+published: True
+---
+
+# Insert Document at Caret Position
+
+The [RadDocumentMerger]({%slug radrichtextbox-features-merge-raddocumentmerger%}) class exposes the **InsertDocument() method** that enables you to merge two [RadDocument]({%slug radrichtextbox-features-document-elements-raddocument%}) instances at the caret position of the targeted one. 
+
+Along with the [RadDocument]({%slug radrichtextbox-features-document-elements-raddocument%}) instance, the overloads of the InsertDocument() method allow you to pass a parameter of type **InsertDocumentOptions**. You can use this parameter to customize the way the merge conflicts are resolved and the new document is inserted. The available properties are as follows:
+
+* **ConflictingStylesResolutionMode**: Determines the mode of resolving conflicts between styles with same IDs. This property is of type ConflictingStylesResolutionMode and could have the following enum values:
+
+	* **UseTargetStyle**: The style of the target document is used. E.g. if the target and source documents have a style with a name "MyStyle", but with different properties, the one from the target is used by the merged document and the one from the source document is not added to the style repository of the target document.
+	
+	* **RenameSourceStyle**: The style of the source document is renamed and used. E.g. if the target and source documents have a style with a name "MyStyle", but with different properties, the one from the source is renamed to "MyStyle\_1", added to the style repository of the target document and used by the merged from the source document elements. *This is the default value*.
+
+* **InsertLastParagraphMarker**: A Boolean property that determines whether the last paragraph marker is inserted and, subsequently, whether the last paragraph formatting is preserved. When the value is true, a new paragraph with the same formatting is inserted. Otherwise, only the inlines from that paragraph are inserted. *The default value is true*.
```

Comment:
You can higlight the word 'true' (ex: bold, italic). 

"When the value is **true**, a new ......"
"The default value is **true**"